Dockerfile

```diff
@@ -98,12 +98,21 @@ RUN set -ex; \
         liblz4-dev \
         libpcre3-dev \
         make \
+        wget \
     '; \
     apt-get update; \
     apt-get install -y $buildDeps --no-install-recommends; \
     rm -rf /var/lib/apt/lists/*; \
     \
     make install-python-dependencies; \
+    mkdir /tmp/uwsgi-dogstatsd; \
+    wget -O - https://github.com/DataDog/uwsgi-dogstatsd/archive/1a04f784491ab0270b4e94feb94686b65d8d2db1.tar.gz \
+        | tar -xvz -C /tmp/uwsgi-dogstatsd --strip-components=1; \
+    uwsgi --build-plugin /tmp/uwsgi-dogstatsd; \
+    rm -rf /tmp/uwsgi-dogstatsd .uwsgi_plugins_builder; \
+    mkdir -p /var/lib/uwsgi; \
+    mv dogstatsd_plugin.so /var/lib/uwsgi/; \
+    uwsgi --need-plugin=/var/lib/uwsgi/dogstatsd --help > /dev/null; \
```

Comment:
is the uwsgi plugin going to try to talk to the datadog agen when we run snuba locally as well ?

Reply:
No, uwsgi needs to be configured to do it. This just installs a .so file in the container fundamentally. We'll configure it in production.

Reply:
Ok, with the latest version, this is baked in that it'll try to emit stats, but nothing is listening and it's all UDP, so shouldn't cause any issues. I'm choosing to explicitly enable this behavior so we can uniformally get this across the board.